### PR TITLE
Unify logging format

### PR DESCRIPTION
**Pull request by aalexandrov**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/gluonts/shell/serve.py [hunk 1]
```diff
@@ -121,7 +122,8 @@ def handle_error(error) -> Tuple[str, int]:
 
     @app.route("/ping")
     def ping() -> Response:
-        app.logger.info("Responding to /ping request")
+        time = datetime.now().isoformat()
+        app.logger.info(f"Responding to /ping request at {time}")
```
Comment (jaheba) [inline src/gluonts/shell/serve.py]: Couldn't we just adapt the logger to also print the time?

**Comment by szha** [comment]
Job PR-216/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-216/1/index.html

File: src/gluonts/shell/serve.py (hunk 1 above)
Comment (aalexandrov) [reply src/gluonts/shell/serve.py]: Done

File: src/gluonts/shell/serve.py [hunk 2]
```diff
@@ -121,7 +127,7 @@ def handle_error(error) -> Tuple[str, int]:
 
     @app.route("/ping")
     def ping() -> Response:
-        app.logger.info("Responding to /ping request")
+        app.logger.info(f"Responding to /ping request")
```
Comment (jaheba) [inline src/gluonts/shell/serve.py]: This can be removed.

**Comment by szha** [comment]
Job PR-216/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-216/2/index.html

**Comment by szha** [comment]
Job PR-216/3 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-216/3/index.html

**Pull request merged**